src/user-data.d.ts

```diff
@@ -36,6 +36,7 @@ type FirestoreSemester = {
 };
 
 type FirestoreSemestersData = {
+  // readonly collections: readonly Collection[]; Note: Not sure where to store collections
```

Comment:
Hmm yes, this would be tricky. Happy to discuss with you and Nidhi sometime at dev sesh or work sesh!